tools/run_tests/performance/run_worker_go.sh

```diff
@@ -17,6 +17,7 @@ set -ex
 
 cd "$(dirname "$0")/../../.."
 
-export GOPATH=$(pwd)/../gopath
+GOPATH=$(pwd)/../gopath
```

Comment:
[SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155)
> Declare and assign separately to avoid masking return values.
> 
> In the original code, the return value of mycmd is ignored, and export will instead always return true. This may prevent conditionals, set -e and traps from working correctly.
> 
> When first marked for export and assigned separately, the return value of the assignment will be that of mycmd. This avoids the problem.

I fixed it as the shellcheck wiki suggested, because it is better than noop.